### DISCUSSION TITLE
[Question][Hw4] score for each part

### DISCUSSION THREAD

**Discussion by frogfrog2** [prosyslab-classroom/cs348-information-security#704] [Q&A]
Name: Jihwan Park

Hello, can you tell me how 190 points are divided into two parts: Acl and Capability?
If I could check if I got a perfect score for the Acl part, it would be helpful for my debugging!
Thanks in advance.

**Comment by sujin0529** [comment]
Hi,
I think it will be difficult to correct this problem right away because I need to fix the scoring machine.
It may be inconvenient, but I will tell you the number of each test case, so could you check it?
-  &#35; of Acl: 7
- &#35; of Capability: 12

Thanks.

**Comment by frogfrog2** [comment]
That's helpful enough for me! Thanks
